settings.gradle.kts

```diff
@@ -105,6 +104,13 @@ dependencyResolutionManagement {
     }
 }
 
+rootProject.name = "kotlinx-atomicfu"
+
+plugins {
+    id("gradle-build-scan")
+    id("gradle-build-cache")
+}
```

Comment:
It's not immediately obvious that these plugins are Atomicfu convention plugins. It'd be nice to give them a descriptive namespace. E.g. add `package atomicfu_build` to the top of the `.kts` files (but keep the files the same location)